preproc_test.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import preproc
+from caffe2.python import workspace, layer_model_helper, schema, core, utils, workspace
+
+features = np.random.rand(10, 2).astype('float32')
```

Comment:
Can you wrap them into a function: test_write_db()? We would like to test each function separately. Then run this test function under "if __name__ == '__main__':".